docs/src/pages/components/Text/AndroidText.mdx

```diff
@@ -11,27 +11,35 @@ import AndroidComponentScreenshots from '../../../components/Screenshots/Android
 import textReadme from '../../../../../backpack-android/docs/Text/README.md';
 import textSpanReadme from '../../../../../backpack-android/docs/TextSpans/README.md';
 
+import TextDefault from '../../../../../backpack-android/docs/Text/screenshots/default.png';
+import TextDefaultDm from '../../../../../backpack-android/docs/Text/screenshots/default_dm.png'
+import TextEmphasized from '../../../../../backpack-android/docs/Text/screenshots/emphasized.png'
+import TextEmphasizedDm from '../../../../../backpack-android/docs/Text/screenshots/emphasized_dm.png'
+import TextHeavy from '../../../../../backpack-android/docs/Text/screenshots/heavy.png'
+import TextHeavyDm from '../../../../../backpack-android/docs/Text/screenshots/heavy_dm.png'
```

Comment:
```suggestion
import TextDefault from 'backpack-android/docs/Text/screenshots/default.png';
import TextDefaultDm from 'backpack-android/docs/Text/screenshots/default_dm.png'
import TextEmphasized from 'backpack-android/docs/Text/screenshots/emphasized.png'
import TextEmphasizedDm from 'backpack-android/docs/Text/screenshots/emphasized_dm.png'
import TextHeavy from 'backpack-android/docs/Text/screenshots/heavy.png'
import TextHeavyDm from 'backpack-android/docs/Text/screenshots/heavy_dm.png'
```

Reply:
tbh I think it's better to just do this inline. Makes it easier to see what's what, and means there's less noise in PR MD previews (as they just omit the components).

Reply:
Actually maybe it's not. I can't decide. Anyway the paths can be simplified regardless 👍

Reply:
Hmm I should've done this already! Will check if I have missing commit

Reply:
Looks like a merge from master messed it up a bit here! Commit incoming